internal/plugincommon/consensus/consensus.go

```diff
@@ -51,13 +51,13 @@ func GetConsensusMapAggregator[K comparable, T any](
 	lggr logger.Logger,
 	objectName string,
 	items map[K][]T,
-	f int,
+	f Threshold,
```

Comment:
Can you make this MultiThreshold as well? I had to change it to make it work with all cases.

Reply:
done